plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergWritableTableHandle.java

```diff
@@ -36,6 +36,7 @@
     private final String outputPath;
     private final FileFormat fileFormat;
     private final Map<String, String> storageProperties;
+    private final boolean retriesEnabled;
```

Comment:
Maybe just pass the RetryMode enum value around?

Reply:
Actually - yeah - makes sense